src/config.rs

```diff
@@ -115,6 +126,13 @@ pub struct BitcoindConfig {
     pub addr: SocketAddr,
 }
 
+/// Everything we need to know for talking to Electrum serenely.
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct ElectrumConfig {
+    /// The IP:port Electrum's RPC is listening on.
+    pub addr: SocketAddr,
```

Comment:
Perhaps we should change this to a string to allow for general URLs?

Reply:
Yes, it could be an enum between either a SocketAddr and a Url to resolve.

Reply:
I wonder if it might be easier to use a single `String` directly. The underlying client uses `Client::new(url: &str)` and handles the case where `url` is prefixed with either "ssl://" or "tcp://" (or neither).

Reply:
For now I've changed it to `String`. We might also want to add other options from `electrum_client::Client`, such as SOCKS5 configuration.

Reply:
I've created https://github.com/wizardsardine/liana/issues/1356 for SOCKS5.